data/hai/hai reveal 4 middle.txt

```diff
@@ -1233,29 +1233,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
+				`	"The Unfettered have offered their help because you would not help your people. If you change your decision, you will not need their involvement."`
+					goto "turn back unfettered"
+			action
+				set "told hai to accept unfettered help"
+			`	One Elder rises and says, "The Unfettered only know how to make war. Once they are in human space, there's no telling what they'll do." Many in the room nod in agreement.`
+			`	Teeneep responds, "They are clever strategists and dedicated in our protection. Never would they take such unnecessary risks. I am certain they would accept an overseer, from among us, to advise them on limits to their actions."`
+				goto "the elders vote"
+			label "turn back unfettered"
+			action
+				set "told hai to reject unfettered help"
+			`	A disgruntled Elder responds, "Change our decision? Our decision not to violate human sovereignty? Even if I lose my Eldership, I will not vote for violence."`
+			`	Teeneep says, "If you take no action, and refuse help from our brethren, they'll pass through our territory anyway. We can hold back Unfettered raids, but their entire armada? That is the unnecessary violence you bring through inaction."`
+			label "the elders vote"
+			`	Another Elder interjects, "You're exaggerating the impact, Teeneep. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone."`
+			`	Many Elders seem appalled by that observation, but the moderator interrupts. "We will vote on two counts: Do we act? Do we accept Unfettered help?"`
+			choice
+				`	"What's going to happen, Teeneep?"`
+					goto teeneep
+				`	"Sayari, what happens next?"`
+			`	Sayari explains, "There's a multi-step voting process, which takes into account dissent between Elders. After each stage of voting, there's discussions between strategically chosen groups of Elders, and afterwards the dissent is usually lower. They iterate until they come to a consensus, unless it takes too long; then they'll use the majority vote."`
+				goto results
+			label teeneep
+			`	Teeneep responds, "They'll vote to do nothing. It's all this council knows how to do." Sayari sighs loudly at this but doesn't contradict Teeneep.`
+			label results
+			`	After a few more minutes of private discussions, the Elders vote "no" on both counts. Your words may not have swayed the vote, but perhaps they'll resonate in the Hai psyche long enough to have an effect.`
+			`	Teeneep is allowed to respond and says ominously, "You know what will happen next."`
+			`	An angry Elder replies, "You wouldn't dare!" Teeneep yells back, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`
+			`	Sayari leads you to a private meeting room.`
```

Comment:
This sequence is gratuitous and feels forced now.

Reply:
I left this awkward part in because an Elder is supposed to threaten Teeneep. I will come up with a different way for the Elder to threaten.